README.md

```diff
@@ -25,7 +32,7 @@ install.packages(
 
 # Continuous maintainer-driven deployment
 
-After the one-time registration process described below, each package in R-releases continuously deploys to R-universe. Every new [tag and release](https://docs.github.com/en/repositories/releasing-projects-on-github/managing-releases-in-a-repository) in your package's GitHub/GitLab repository automatically publishes at <https://r-releases.r-universe.dev> and becomes available to install from there. No matter how many releases you create in your own personal GitHub account, you do not need to interact with R-releases again unless you wish to change the link or remove the package from <https://r-releases.r-universe.dev>. In other words, you as the maintainer are in complete control. This painless maintainer-driven experience was made possible by the incredible infrastructure of [rOpenSci's R-universe system](https://ropensci.org/r-universe/) and insights from [Jeroen Ooms](https://github.com/jeroen/).
+Each package only needs to register once. After registration, every new [release](https://docs.github.com/en/repositories/releasing-projects-on-github/managing-releases-in-a-repository) automatically deploys to <https://r-releases.r-universe.dev> without needing to go through R-releases again. In other words, the package maintainer is in complete control. This painless maintainer-driven experience was made possible by the incredible infrastructure of [rOpenSci's R-universe system](https://ropensci.org/r-universe/) and insights from [Jeroen Ooms](https://github.com/jeroen/) and [Charlie Gao](https://github.com/shikokuchuo/).
```

Comment:
I still don't think it's appropriate to credit me for this particular point, although appreciate the sentiment!

Reply:
Fair enough, removed in 551f11d280f874065e3c7675fd4e441425fcd3f8